### DISCUSSION TITLE
I have problems switching my project from Local-Mode to Prod-Mode

### DISCUSSION THREAD

**Discussion by jucegt** [tinacms/tinacms#3675] [Help] [answered]
Hello

Recently, I started a project with Tina, in development mode everything works fine(I am working with separate repositories) but when I try to deploy my project to Vercel I get the following error:

![Screenshot 2023-03-07 at 5 39 08 PM](https://user-images.githubusercontent.com/5516040/223580478-22bc569c-4420-4c2b-a1b3-e60fd5f12041.png)

When I do a local build everything works perfectly:

![Screenshot 2023-03-07 at 5 28 51 PM](https://user-images.githubusercontent.com/5516040/223580862-525fa5aa-f4c6-4eef-81a4-8cd53c65beff.png)

This is the config.ts file of Tina:

```ts
import { defineConfig } from 'tinacms';

const branch =
  process.env.HEAD || process.env.VERCEL_GIT_COMMIT_REF || 'master';
const clientId = process.env.NEXT_PUBLIC_TINA_CLIENT_ID || null;
const token = process.env.TINA_TOKEN || null;
const localContentPath =
  process.env.VERCEL === undefined ? process.env.LOCAL_CONTENT : undefined;

export default defineConfig({
  branch,
  clientId,
  token,
  localContentPath,
  build: {
    outputFolder: 'admin',
    publicFolder: 'public',
  },
  schema: {
    collections: [
       // My collections
    ],
  },
});
```

Does anyone have an idea what might be going on? What am I doing wrong?

**Comment by jamespohalloran** [comment, marked as answer]
We just released a fix in @tinacms/cli 1.3.1 for this. There might still be an underlying issue (such as your updated .tina files not being pushed up), but we should now be surfacing a clearer error if that's the case!

**Reply by jucegt** [reply]
Thank you for your help

I have updated the version of @tina/cli and it works perfect, my site is already deployed and working perfectly.
